README.md

```diff
@@ -75,15 +41,59 @@ The distances can also be used to produce a dendrogram, showing the result of hi
 - PyYAML
 - SciPy
 
-CBI and its dependencies can be installed using `setup.py`:
+
+## Installation
+
+The latest release of CBI is version 1.2.0. To download and install this
```

Comment:
I think I prefer "release" here.  I tend to says like "we may do this in future releases" as opposed to "in future versions", and the security documentation also talks about "releases" instead of "versions".